api/clients/v2/payload_disperser.go

```diff
@@ -146,13 +146,20 @@ func (pd *PayloadDisperser) SendPayload(
 	}
 	pd.logger.Debug("Payload encoded to blob")
 
-	timeoutCtx, cancel := context.WithTimeout(ctx, pd.config.DisperseBlobTimeout)
+	timeoutCtx, cancel := context.WithTimeout(ctx, pd.config.ContractCallTimeout)
+	defer cancel()
+	requiredQuorums, err := pd.certVerifier.GetQuorumNumbersRequired(timeoutCtx, certVerifierAddress)
```

Comment:
this works but isn't so optimal since this is additional latency added to the dispersal wire. Curious if we should introduce a cache that persists `verifier_address:quourms`